app/uk/gov/hmrc/teamsandrepositories/controller/model/Repository.scala

```diff
@@ -34,9 +35,10 @@ case class Repository(
 
 object Repository {
 
-  def create(gr: GitRepository): Repository =
+  def create(gr: GitRepository, teamNames: Seq[String] = Nil): Repository =
```

Comment:
https://blog.ssanj.net/posts/2019-05-01-why-are-default-parameter-values-considered-bad-in-scala.html